test/features/buckets/key_value_bucket.js

```diff
@@ -1,4 +1,6 @@
-"use strict";
+'use strict';
+
+/* global describe, it, before */
```

Comment:
Actually, you can avoid doing that by placing a test-specific eslint config into the root of the test folder - I think it's much nicer like this. 

Example: https://github.com/wikimedia/preq/blob/master/test/.eslintrc.yml